lib/urbanopt/geojson/schema/building_properties.json

```diff
@@ -6,35 +6,18 @@
   "type": "object",
   "properties": {
     "id": {
```

Comment:
Is there a way to add a comment/field for the version of the file?

Reply:
I don't know of a good way to do this, I think the best thing is to use the version of the gem